app/src/main/java/com/harman/wearosapp/app/hr_service/HRBroadcastReceiver.kt

```diff
@@ -0,0 +1,36 @@
+package com.harman.wearosapp.app.hr_service
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import androidx.health.services.client.data.DataType
+import androidx.health.services.client.data.PassiveMonitoringUpdate
+import com.harman.wearosapp.domain.use_cases.HRUseCase
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.get
+
+class HRBroadcastReceiver() : BroadcastReceiver(
```

Comment:
I think it is better to move this to data layer

Reply:
done